### ISSUE TITLE
New version 4.5 is coming in middle November

### ISSUE THREAD

**Issue by mistertest** [mistertest/xcloud-shield#42]
Hello every one. 
Firstly I want to thank you for all your support and messages. You are now so much to use my unofficial Xcloud shield App. I'm  in a new city from september ago for a new developer job. My code is not with me, but In november I will have access to it, and I will having some time for working on it.

I can annonce you, that the new version 4.5 is comming for middle November.

This version will being the same of the interface and features of Xcloud on Samsung TV and will giving certainly a better global experience in game too. 

I hope that this new version will giving a better support of controller, because this is already the case with the  Samsung tv. So I can think that we'll have the same benefits. 

https://news.xbox.com/en-us/2022/06/09/xbox-app-for-smart-tvs/
So I will need some testers for my app in this time period.

See you later, my friends.

**Comment by Xabiman** [comment]
Thank you very much, friend. It would be a pleasure to be a tester, of course.

**Comment by Potiron** [comment]
Thanks you! You rocks!
Waiting for your upgrade and will hopefully include than ’edge enhanced quality’ feature too 😄

I can test dual sense and other pad as well,like microsoft one ans others

**Comment by Xabiman** [comment]
I can test PS5 and PS4 bluetooth controllers.

**Comment by ExSport** [comment]
I have PS3 and latest XBOX controllers for testing...
Btw. @mistertest hopefully you will not disappear again for another 8 months after the post beta version will be available "next week" (language issues, 720p version, etc... mentioned in other posts) 😄
Thank you 👍

**Comment by mistertest** [comment]
> I have PS3 and latest XBOX controllers for testing... Btw. @mistertest hopefully you will not disappear again for another 8 months after the post beta version will be available "next week" (language issues, 720p version, etc... mentioned in other posts) 😄 Thank you 👍

Thank You ExSport. 
Yes lol, I'm sure, I have now more times for the project.

**Comment by DVergil** [comment]
Eu gostaria de testar.
O meu fire stick lite não permite eu usar o aplicativo de mapear os controles
Será que tem como contornar isso?

**Comment by Tracker997** [comment]
I would also like to test it! :)

**Comment by mister-callisto** [comment]
Bonjour Mistertest,
c'est avec plaisir que je me pote volontaire pour être beta testeur de cette nouvelle distribution :)
Cordialement,

**Comment by Shizzl1** [comment]
Hi, if needed I can test too, nvidia shield 2015 with 2017 shield controller. Thx for ur work.

**Comment by shashanknimje** [comment]
Please let me know if you need another tester, happy to test with Nvidia Shield 2017 (Model: P2897, Version: 11) and Xbox Series Controller. And thank you for all your work! :)

**Comment by danfozzy** [comment]
2019 pro user here, with shield controller (2017), DS4 and Xbox one controller.

Happy to help if needed

**Comment by DVergil** [comment]
Ok

**Comment by danfozzy** [comment]
> Hello every one. Firstly I want to thank you for all your support and messages. You are now so much to use my unofficial Xcloud shield App. I'm in a new city from september ago for a new developer job. My code is not with me, but In november I will have access to it, and I will having some time for working on it.
> 
> I can annonce you, that the new version 4.5 is comming for middle November.
> 
> This version will being the same of the interface and features of Xcloud on Samsung TV and will giving certainly a better global experience in game too.
> 
> I hope that this new version will giving a better support of controller, because this is already the case with the Samsung tv. So I can think that we'll have the same benefits.
> 
> https://news.xbox.com/en-us/2022/06/09/xbox-app-for-smart-tvs/ So I will need some testers for my app in this time period.
> 
> See you later, my friends.

The Samsung TV app for Xbox game pass works really well with 4k setups, is there any hope this is the case for the update or will it be based on webview?

**Comment by mistertest** [comment]
Hello every one.I'm working on the project, all the weekend and watching the World Cup, on the same time. I'm French so I'm supported the team of Mbape, but my parents are from Moroco so I'm supported the Maroc too.
I think that the French team has the real potential to being the Champions one more time 😄

**Comment by mistertest** [comment]
@danfozzy The games are not running on 4K with the samsung tv xcloud app. For geforce now yes just recently, but not for xcloud yet. But the support and uspscale is working certainly very well. My app is always based from a webview, so the upscale will not working. I will have some help from other developers with new objectives and working about this questions.

**Comment by DVergil** [comment]
I'm willing to test

Em sáb, 26 de nov de 2022 17:39, mistertest ***@***.***>
escreveu:

**Comment by DVergil** [comment]
I will test on fire stick lite

Em sáb, 26 de nov de 2022 18:13, Danilo Alencar ***@***.***>
escreveu:

**Comment by DVergil** [comment]
Good luck

**Comment by pums974** [comment]
I'm also willing to test (nvidia shield)

**Comment by danfozzy** [comment]
> @danfozzy The games are not running on 4K with the samsung tv xcloud app. For geforce now yes just recently, but not for xcloud yet. But the support and uspscale is working certainly very well. My app is always based from a webview, so the upscale will not working. I will have some help from other developers with new objectives and working about this questions.

Yep, it's not upscaled to 4k but it plays a hell of alot better on the Samsung app on the 4k TV, Vs the shield using either of the official or this app on the same tv. Would love to know how they do it 😂

**Comment by Alumun** [comment]
Hi, I have an Android TV box with Amlogic S905X4 it's a 4x ARM Cortex-A55 @ 1.91 GHz with 2GB RAM, android TV 10. 

The app stay in a Blank screen, I have the WebView updated

**Comment by danfozzy** [comment]
Hey there, no pressure to answer but is there any update, anything that we can help with?

**Comment by DVergil** [comment]
Update?

**Comment by Potiron** [comment]
> Hi, I have an Android TV box with Amlogic S905X4 it's a 4x ARM Cortex-A55 @ 1.91 GHz with 2GB RAM, android TV 10.
> 
> The app stay in a Blank screen, I have the WebView updated

Almost the same processeur in my xiaomi mi tv p3 and same issue

**Comment by ExSport** [comment]
> > I have PS3 and latest XBOX controllers for testing... Btw. @mistertest **_hopefully you will not disappear again for another 8 months_** after the post beta version will be available "next week" (language issues, 720p version, etc... mentioned in other posts) 😄 Thank you 👍
> 
> Thank You ExSport. Yes lol, I'm sure, I have now more times for the project.

@mistertest It was brave to promise anything in the run-up to Christmas/Qatar.😊
Happy holidays and no stress!👌

**Comment by kdejaeger** [comment]
It's vacation time I guess :). BTW, I did this discovery last week: https://www.reddit.com/r/ShieldAndroidTV/comments/znahae/magic_trick_to_work_around_stutterlag_in_xbox/

**Comment by pums974** [comment]
Any news? Is it possible to help ?

**Comment by danfozzy** [comment]
really hoping this project hasn't been shelved :( @mistertest is there anything we can do to help out?

**Comment by razfaz23** [comment]
Same here. I just found this recently as I  just got a Shield TV Pro myself. Looks promising, and would like to support if this is still being developed. Thanks for what you have accomplished so far.

**Comment by ExSport** [comment]
Seems **dead** again 😑

**Comment by pums974** [comment]
Maybe @mistertest you could publish the sources so someone could help you

**Comment by Hatsu5673** [comment]
Volta com o projeto mano

**Comment by mister-callisto** [comment]
unfortunately the creator of this project died more than a year ago. His work will forever remain unfinished.
